app/src/main/java/com/android/shelfLife/MainActivity.kt

```diff
@@ -11,6 +11,9 @@ import androidx.navigation.compose.composable
 import androidx.navigation.compose.rememberNavController
 import androidx.navigation.navigation
 import com.android.shelfLife.model.camera.BarcodeScannerViewModel
+import com.android.shelfLife.model.foodFacts.FoodFactsRepository
```

Comment:
Nit: Please ensure that any unused imports are removed before merging into the main branch to keep the code clean and avoid unnecessary clutter.